src/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+set(TARGET piTest)
+set(SOURCES
+	piTest.c
+	piControlIf.c
+)
+
+add_executable(${TARGET} ${SOURCES})
+target_include_directories(${TARGET} PRIVATE ../inc ../lib/piControl)
```

Comment:
As I can't comment on the file rename I do it here. Please use include not inc as directory name.